kubernetes/readme.md

```diff
@@ -0,0 +1,18 @@
+#Kubernete Setup
+
+```
+#create secrets
+kubectl create secret generic todo-secrets --from-literal=DB_URL=xxxxx
```

Comment:
It would be nice to extend this to include all the secrets, e.g.
```suggestion
kubectl create secret generic todo-secrets --from-literal=DB_URL=xxxxx --from-literal=CLIENT_ID=xxxxx --from-literal=CLIENT_SECRET=xxxxx --from-literal=LOGGLY_TOKEN=xxxxx 
```